bahmni_api_feed/models/order_save_service.py

```diff
@@ -488,7 +488,8 @@ def _get_product_ids(self, order):
         if order['productId']:
             prod_ids = self.env['product.product'].sudo().search([('uuid', '=', order['productId'])])
             if not prod_ids:
-               raise UserError ("Product does not exits in ERP or Inactivated in ERP master.")
+                #log message indicating this product/service was skipped, because it wasn’t found in Odoo.
+               _logger.warning("Product does not exist in ERP or Inactivated in ERP master.- %s(%s)"%(order['productName'],order['productId']))
```

Comment:
Would be great if we can avoid using ERP and maybe have the warning say something generic like - "[productId] [productName] does not exists or is inactive in master"

Reply:
@mohan-13 , Is this change needed?

Reply:
@hari1119 The change Rahul suggested also makes sense, pls refactor the message accordingly

Reply:
Okey sure